update_rate: 3  #Hz
y_axis_swap: false
map_height: 200 # pixel
map_width: 200 #pixel
map_resolution: 0.1 #pixel to meter
density_img_dir: /home/kangneoung/sw_repo/decent_voronoi/src/decent_voronoi/test/density_new/case2/wall.png
max_agent_num: 10
is_hete_cov_radius: true
is_dropout_use: false
is_propa_connected_area: true  # true -- propagate to only visible area, false -- propagate regardless of visibility
text_log_enable: true
text_log_dir: /home/kangneoung/sw_repo/decent_voronoi/src/decent_voronoi/test/log/case2_1_hete_unity/
img_save_dir: /home/kangneoung/sw_repo/decent_voronoi/src/decent_voronoi/test/opt_animation/
img_color_label_dir: /home/kangneoung/sw_repo/dynamic_voronoi/src/dynamic_voronoi/label/label.txt
